package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@okta/okta-signin-widget",
-  "version": "1.13.0",
+  "version": "2.1.0",
```

Comment:
This was regenerated when we updated the dependencies. Current version is `2.1.0`